CHANGELOG.md

```diff
@@ -24,6 +24,8 @@ the code was deployed.
 - Low battery checks in heartbeat api for firmware state machine (CU-4avrhp).
 - Tracking of reset reason to reconnection alerts (CU-z9e1md).
 - Heartbeat messages enriched, including state transition data (CU-1c1qd5u).
+- `/firmware` folder with production code for Boron devices (CU-un6ybt).
```

Comment:
my initial instinct was for only the existing production version to be included in there. Siren is a bit weird because it's not legacy code, nor is it slated for production - but given that it's merged into the server code... I think you're right and i should put it in there.

For Argons, my understanding is that the code is deprecated/legacy until and if we run into the case where Boron's don't work. If that happens we go to the old repo and fish it out to integrate it